app/Entry.js

```diff
@@ -40,7 +40,7 @@ import { ENLocalDiagnosisKeyScreen } from './views/Settings/ENLocalDiagnosisKeyS
 import { FeatureFlagsScreen } from './views/FeatureFlagToggles';
 import ImportScreen from './views/Import';
 import { EnableExposureNotifications } from './views/onboarding/EnableExposureNotifications';
-import Onboarding1 from './views/onboarding/Onboarding1';
+import Welcome from './views/onboarding/Welcome';
```

Comment:
I like this name change and will do the same for the other onboarding screens so they're more descriptive of the actual screen function